### PR TITLE
cortex m0plus example

### PR DESCRIPTION
**Pull request by joextodd**
- Update README
- Always use `chirp` for the SDK pointer, as in the documentation
- Add example for MKRZero using the same style as other examples, highlighting that it can also be used for other boards.

Closes: #3

### CODE REVIEW

File: examples/MKRZeroSend/MKRZeroSend.ino [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+  i2s.begin(I2S_32_BIT, SAMPLE_RATE);
+  i2s.enableMCLK();
+  i2s.enableTx();
+
+  size_t payload_len = 5;
+  uint8_t *payload = chirp_connect_random_payload(chirp, &payload_len);
+
+  char *hex = chirp_connect_as_string(chirp, payload, payload_len);
+  Serial.print("Generated payload: ");
+  Serial.println(hex);
```
Comment (LittleMonsterFr) [inline examples/MKRZeroSend/MKRZeroSend.ino]: free the `hex` pointer :)

File: examples/MKRZeroSend/MKRZeroSend.ino [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+// Chirp -----------------------------------------------------
+
+void chirpErrorHandler(chirp_connect_error_code_t code)
+{
+  if (code != CHIRP_CONNECT_OK)
+  {
+    const char *error_string = chirp_connect_error_code_to_string(code);
+    Serial.println(error_string);
+    chirp_connect_free((void *) error_string);
```
Comment (LittleMonsterFr) [inline examples/MKRZeroSend/MKRZeroSend.ino]: no need to free this one

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 Recent changes to the [Chirp Arduino SDK](https://developers.chirp.io/docs).
 
+## v3.3.0 (beta)
+ - Added send-only support for cortex-m0plus (MKRZero, MKR Vidor 4000)
+ - Added support for cortex-m4 (Nano 33 Sense)
+ - Build v3.2.10
```
Comment (LittleMonsterFr) [inline CHANGELOG.md]: Don't know if i wont update the build number to something like 3.3.0-rc1 as there are a few important changes + this is a kind of beta release

Comment (joextodd) [reply CHANGELOG.md]: Agreed

File: README.md [hunk 4]
```diff
@@ -23,45 +42,51 @@ Install ChirpSDK as a library. For instructions, see
...
 
-To set up the Chirp SDK, initialise and configure with your app key, secret and `arduino` config from the [Developer Hub](https://developers.chirp.io). Then set any required callbacks and start the SDK running.
+To set up the Chirp SDK, initialise and configure with your app key,
+secret and config from the [Developer Hub](https://developers.chirp.io).
 
-    connect = new_chirp_connect(APP_KEY, APP_SECRET);
-    if (connect == NULL) {
+*Note* that any configuration will work for the send only boards, however it is
+recommended to use a lightweight protocol such as `16khz-mono` when receiving data on
+Arduino due to the minimal memory available.
```
[the remark this answers is not in the document]
Comment (joextodd) [reply README.md]: This does need amending, as we will want everyone to use the `arduino` protocol. I will make this change once the protocol has been finalised. 

However this statement I think is true. We don't get support ultrasonic receiving on Arduino...

[the remark this answers is not in the document]
Comment (LittleMonsterFr) [reply README.md]: It doesn't work but that's not a memory issue - 256k is plenty of memory 😛 But feel free to keep this, it was more to highlight it than change it

[the remark this answers is not in the document]
Comment (joextodd) [reply README.md]: Ah I see, yes I'll reword it

File: CHANGELOG.md (hunk 3 above)
Comment (LittleMonsterFr) [reply CHANGELOG.md]: I meant for the c-sdk build 🙈

File: README.md [hunk 5]
```diff
@@ -44,18 +44,17 @@ Once installed, you can access the example programs from the menu :
 
 ```File > Examples > ChirpSDK > Example ```
 
-and you can include the headers to use Chirp in your own code by using :
+and you can include the headers to use Chirp in your own code by adding :
 
-```Sketch > Import Library > ChirpSDK```
+```#include "chirp_connect.h"```
```
Comment (LittleMonsterFr) [inline README.md]: Don't you need to add the library anyway ?

Comment (joextodd) [reply README.md]: Nope Arduino locates it automatically somehow. In fact when you use the `Import Library`, all it does is include all of the unnecessary chirp header files.

Comment (joextodd) [reply README.md]: Worth double checking though

Comment (LittleMonsterFr) [reply README.md]: 👍 👍 👍 👍

**Pull request merged**